src/onefetch/cli.rs

```diff
@@ -172,12 +173,18 @@ impl Cli {
                 .multiple(true)
                 .takes_value(true)
                 .help("Ignore all files & directories matching EXCLUDE."),
+            )
+        .arg(
+            Arg::with_name("off")
+                .long("off")
```

Comment:
Maybe we should add `.conflicts_with_all(&["image", "ascii-language", "ascii-input"])`